src/modules/ndi/consumer_ndi.c

```diff
@@ -280,6 +255,11 @@ static int consumer_ndi_start( mlt_consumer consumer )
 
 	mlt_properties properties = MLT_CONSUMER_PROPERTIES( consumer );
 
+	// Set progressive/interlacing properties
+	mlt_profile profile = mlt_service_profile( MLT_CONSUMER_SERVICE( consumer ) );
+	mlt_properties_set_int( properties, "top_field_first", !profile->progressive );
+	mlt_properties_set_int( properties, "progressive", profile->progressive );
```

Comment:
FYI, This is already taken care of in mlt_consumer.c apply_profile_properties(), which is called inside mlt_consumer_start(), but it does not hurt anything to do it here as well.